tests/expand/st04a-in.jsonld

```diff
@@ -0,0 +1,8 @@
+{
+  "@id": "ex:s",
+  "ex:p": {
+    "@reifies": {
+      "ex:prop": "value"
+    }
+  }
+}
```

Comment:
why not simply this?
```suggestion
{
  "@id": "ex:s",
  "@reifies": {
    "ex:prop": "value"
  }
}
```
(which would also change the expanded version, of course...

Reply:
Generally, tests such as st04a are based on st04 and just replace `@triple` with `@reifies`; there are a bunch of these, so it was generally easier to do this, even though it uses an unnecessary indirection through a predicate and new node object. But, I can go and change all these, which exist separately in pretty much each test manifest.